### PR TITLE
Added RHAI Overview

### PR DESCRIPTION
**Pull request by quarckster**
Added RHAI Overview entity and the respectful views.

### CODE REVIEW

File: airgun/views/rhai.py [hunk 1]
```diff
@@ -10,3 +10,20 @@ class AllRulesView(BaseLoggedInView):
...
+
+class OverviewDetailsView(BaseLoggedInView):
+    title = Text(".//h1[normalize-space(.)='Overview']")
+    inventory = Table(".//h2[span[normalize-space(.)='Newest Systems']]/"
+                      "parent::section//table")
+    security_issues = Text(".//div[@ng-if='securityErrors']/div")
+    stability_issues = Text(".//div[@ng-if='stabilityErrors']/div")
+
+    @property
+    def is_displayed(self):
```
Comment (oshtaier) [inline airgun/views/rhai.py]: that logic is invalid as I opened the page and don't have `security_issues` and `stability_issues` elements on the page, so doubtfully they are criteria for `is_displayed`

File: airgun/views/rhai.py [hunk 2]
```diff
@@ -10,3 +10,20 @@ class AllRulesView(BaseLoggedInView):
...
+        return (
+            self.title.is_displayed and
+            self.security_issues.is_displayed and
+            self.inventory.is_displayed and
+            self.stability_issues
```
Comment (oshtaier) [inline airgun/views/rhai.py]: probably wrong copy/paste. Anyway is not needed

File: airgun/views/rhai.py [hunk 3]
```diff
@@ -10,3 +10,20 @@ class AllRulesView(BaseLoggedInView):
     @property
     def is_displayed(self):
         return self.title.is_displayed and self.search.is_displayed
+
+
+class OverviewDetailsView(BaseLoggedInView):
+    title = Text(".//h1[normalize-space(.)='Overview']")
+    inventory = Table(".//h2[span[normalize-space(.)='Newest Systems']]/"
```
Comment (oshtaier) [inline airgun/views/rhai.py]: `//section[contains(., 'Newest Systems')]//table` sounds better

File: airgun/entities/rhai/overview.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep, navigator
+from airgun.views.rhai import OverviewDetailsView
+
+
+class OverviewEntity(BaseEntity):
+
+    @property
```
Comment (oshtaier) [inline airgun/entities/rhai/overview.py]: do simple def read()
view = self.navigate_to(self, "Details")
return view.read()

and then in test just assert necessary values as a strings or transform then in int on the fly if it is absolutely necessary

**Comment by quarckster** [comment]
@oshtaier changes made

**Pull request merged**